redaxo/src/addons/install/lang/de_de.lang

```diff
@@ -35,6 +35,8 @@ install_send = Senden
 install_upload_file = Zip-Datei erstellen und hochladen
 install_replace_assets = Assets aus dem Root-Assets-Ordner nehmen
 install_ignore_tests = Ordner "tests" ignorieren
+install_release = Release
```

Comment:
„Releasedatum“ oder ggf. „Veröffentlichungsdatum“ oder „Veröffentlicht am“ ?

Reply:
ich hatte mich gegen "Veröffentlichungsdatum" entschieden, da das Wort einfach doppelt so viel platz wie das eigentliche Release-Datum in Anspruch nimmt...
Vielleicht wäre dann "Releasedatum" besser geeignet.

Reply:
@staabm so kurz wie möglich um schmale Spalten zu behalten. Evtl. Veröffentlicht und englisch released

Reply:
oder sogar nur Datum / date. da es sich ja eh auf die Version bezieht

Reply:
Mir ist es nicht so wichtig. Aber „Release“ alleine finde ich auf den 1. blick unpassend.

Ggf. Ist es im context mit der tabelle aber selbsterklärend?